app/src/main/java/com/eveningoutpost/dexdrip/LibreReceiver.java

```diff
@@ -69,7 +69,7 @@ public void run() {
                                 Libre2RawValue currentRawValue = processIntent(intent);
                                 if (currentRawValue == null) return;
                                 Log.v(TAG,"got bg reading: from sensor:"+currentRawValue.serial+" rawValue:"+currentRawValue.glucose+" at:"+currentRawValue.timestamp);
-                                if(!BgReading.last_within_minutes(5)) {
+                                if(!BgReading.last_within_millis(270000)) {
```

Comment:
I usually write this as `4.5 * 60 * 1000` or `4.5 * 60000` to make it easier to read.

Reply:
Good idea, will change it.

Reply:
Done.

Reply:
`4.5 * MINUTE_IN_MS` also makes it easier.

Reply:
The travis check went wrong after changing the ms calculation. Any idea?

Reply:
last_within_millis() only accepts LONG. Calculations with a fraction results in an conversion error.

Reply:
The current travis check for 45 * 6 * 1000 still runs for 20 min++ now.

Reply:
and failed again. I have absolute no idea what went wrong here.

Reply:
Unfortunately just ignore travis errors at the moment, as far as I can work out part of fabric's cloud is down causing build failures within the wear module. Its unrelated to this PR.

Reply:
OK. Anything I can do? Is the PR usable in that form?

Reply:
If you are convinced that you have tested these changes and it is working then I can approve it.

Reply:
The changes are live tested and well working. Please approve.